pallets/nft-manager/src/lib.rs

```diff
@@ -808,8 +863,12 @@ impl<T: Config> Pallet<T> {
         unique_external_ref: &BoundedVec<u8, NftExternalRefBound>,
     ) -> DispatchResult {
         ensure!(unique_external_ref.len() > 0, Error::<T>::ExternalRefIsMandatory);
+        let unique_reference = WeakBoundedVec::<u8, NftExternalRefBound>::force_from(
```

Comment:
isn't this part of another pr related to bounded vec? Maybe you should rebase and review your changes